weaviate/collection/classes/internal.py

```diff
@@ -37,3 +42,12 @@ def _metadata_from_dict(metadata: Dict[str, Any]) -> _MetadataReturn:
         score=metadata.get("score"),
         is_consistent=metadata.get("isConsistent"),
     )
+
+
+def _extract_props_from_list_of_objects(type_: Any) -> Optional[Any]:
```

Comment:
no usage?

Reply:
Relic of large PR, it will come back later but will delete for now :)